app/scripts/views/home/home.js

```diff
@@ -3,19 +3,75 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 define([
+  'jquery',
+  'fullpage',
   'views/base'
-], function (BaseView) {
+], function ($, fullpage, BaseView) {
   'use strict';
 
   var HomeView = BaseView.extend({
+    events: {
+      'click .skip-intro':'_skipIntro',
+      'click .arrow': '_moveDown'
+    },
+
     initialize: function () {
-      this.$('h2').html('home sweet home');
+      this.mediaNames = ['video', 'story', 'photo', 'article', 'song', 'picture'];
+      this._setRandomColors();
+      this._initializeFullpage();
     },
 
     // the html is delivered in the page by the server. nothing to render.
     render: function () {
       return this;
+    },
+
+    _initializeFullpage: function() {
+      var that = this;
+      $('#landing-page-wrapper').addClass('is-fade-in');
+      $('#landing-page').fullpage({
+        navigation: true,
+        navigationPosition: 'right',
+        onLeave: function(index, nextIndex, direction){
+          if (nextIndex === 2) {
+            that._swapContent();
+          }
+        }
+      });
+
+    },
+
+    _setRandomColors: function() {
```

Comment:
I guess this is fine for now, but it seems like we probably want to pick a set of colors to use at each step, rather than use a random color unrelated to chronicle's palette